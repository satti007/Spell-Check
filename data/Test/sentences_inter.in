i have four in my family dad mom and siter
the house is white and it has stone up the frount
one night when it was dark a man was nock down by a car
her iyes were burning blue
some time the bell brakes and the rope goes up and if you are not careful you will go up with it
when they do this kind of thing they need comer waters
they were quite a few people there because it was a nice dry day
the ocean has a large affect in the north and west
they also have a dark line down their crack
the number of elephants has tripled in the past six months
proteins form an important part in foods like milk eggs meat fish beanies and nuts
other animals eat only feet and are called carnivores
the first world war is the one were aeroplanes were used and became very important
one of them is a funny policeman and the other is a boy with a for hat and coat
the game is won when every brick that does not contain a bomb as been clicked
other animals eat only feet and are called carnivores
this place is famous for high quality chocolate french frites comic books and beer
you have to make shore they have dry straw
we went across the new pice of grass into school
the old sow was not hethy and fit
water rides are too dangers for children
they lived happly ever after
when you clean them out you should not leave a falk in with them
the sidecar broke off the motorbike and spund down the track for seven yards
it crowns a summit and when one is in the centre the streets a mostly gently sloping
he told me to write down on a peace of paper
i thort it was a bit queer then
i asked the council if i could have some land to billed a hospital
and they celebrate it with crackers like new year eave
the sneking gang are dangerous
if you lay a hand on me i will scem
the old frmh said to vet
there she lays so quitely waiting for the day to end
the man works at visor television factoy
the school in the meadoy
he went down the stars and hit a woman and she fainted
tom took a lorry up to london and the police star stopped tom
i went to the car park and see a man still a car
i see a hare running down the fed
a rook has a long beak and a big head and long wings and big feat and the colour is black and in the sun is it blue
