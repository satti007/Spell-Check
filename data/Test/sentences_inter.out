siter	sister
frount	front
nock	knocked
iyes	eyes
brakes	breaks
comer	calmer
they	there
affect	effect
crack	back
past	last
beanies	beans
feet	meat
were where
for	fur
as	has
feet	meat
frites	fries
shore	sure
pice	piece
hethy	healthy
dangers	dangerous
happly	happily
falk	fork
spund	spun
a are
peace	piece
thort	thought
billed	build
eave	eve
sneking	sneaking
scem	scream
frmh	farmer
quitely	quietly
factoy	factory
meadoy	meadow
stars	stairs
star	car
still	steal
fed	field
feat	feet
